presto-hive/src/test/java/com/facebook/presto/hive/TestHiveFileFormats.java

```diff
@@ -941,14 +942,14 @@ public FileFormatAssertion isReadableByRecordCursor(HiveRecordCursorProvider cur
             return this;
         }
 
-        public FileFormatAssertion isFailingForPageSource(HiveBatchPageSourceFactory pageSourceFactory, HiveErrorCode expectedErrorCode, String expectedMessage)
+        public FileFormatAssertion isFailingForPageSource(HiveBatchPageSourceFactory pageSourceFactory, ErrorCodeSupplier expectedErrorCode, String expectedMessage)
```

Comment:
and perhaps change these to accept `presto.spi. ErrorCode`

that's just a different approach, not sure if better or worse